tests/Fixtures/Mutations/InviteUserMutation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Bakery\Tests\Fixtures\Mutations;
+
+use Illuminate\Support\Str;
+use Bakery\Support\Arguments;
+use Bakery\Mutations\Mutation;
+use Bakery\Types\Definitions\Type;
+use Bakery\Tests\Fixtures\Models\User;
+
+class InviteUserMutation extends Mutation
+{
+    /**
+     * Define the type of the RootField.
+     *
+     * @return Type
+     */
+    public function type(): Type
+    {
+        return \Bakery\Type::boolean();
```

Comment:
This typing issue is a bit annoying so we should see if we can do something about it.